ateam_ui/src/src/state.ts

```diff
@@ -128,6 +135,43 @@ export class AppState {
             });
     }
 
+    setIgnoreFieldSide(ignore_side: number) {
+        const state = this; // fix dumb javascript things
+
+        let int_side = 0;
+        if (ignore_side > 0) {
+            int_side = 1;
+        } else if (ignore_side < 0) {
+            int_side = -1;
+        }
+        const request = new ROSLIB.ServiceRequest({
+            ignore_side: int_side
+        });
+
+        this.services["setIgnoreFieldSide"].callService(request,
+            function(result) {
+                if(!result.success) {
+                    console.log("Failed to set ignore side: ", result.reason);
```

Comment:
Should probably be `console.error` or `console.warn`
Same with the other instances of logging unsuccessful requests